views/js/pciCreator/ims/mathEntryInteraction/creator/widget/states/Map.js

```diff
@@ -73,48 +78,136 @@ define([
     }
 
     MathEntryInteractionStateResponse.prototype.initForm = function initForm() {
-        var self = this,
-            interaction = self.widget.element,
-            $responseForm = self.widget.$responseForm;
+        var interaction = this.widget.element;
+        var $responseForm = this.widget.$responseForm;
 
+        var response = interaction.getResponseDeclaration();
+        var mapEntries = response.getMapEntries();
+        var mappingDisabled = _.isEmpty(mapEntries);
```

Comment:
**chore:** ES6+ is now enabled for PCI, `const` and `let` should be used in place of `var`.

Reply:
yes, I want to refactor once everything works. I have changed so much that now all code is mixed, sorry about that.